kedro-org/kedro-viz
# Chore/utility-bar-icons-animation

## Description

Fixes #930 

## Development notes

![ezgif com-gif-maker (4)](https://user-images.githubusercontent.com/32060364/177419026-d12dddf2-3f78-4d58-a9f5-84b72e0ea244.gif)

## QA notes

- I currently apply 300 ms as a duration for the fade-in/out effect, does it look okay? Any suggestion @GabrielComymQB @Mackay031 ?

- Also, I currently put `sliding-animation.js` in the same location as `experiment-primary-toolbar.js`, I agree this is not the right place but I'm not 100% sure where would best the best place for this? Let me know if you think it should belong somewhere else @tynandebold @rashidakanchwala @cvetankanechevska 

- I also use react-transition-group which is also installed for kedro-viz but have not been used anywhere. I personally like to use it as it helps to manage all the different states, and if we're thinking of having it as a reusable component somewhere else, it's also possible to implement. 


## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/961"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
